FontMenuNameDB

```diff
@@ -7,146 +7,79 @@
 
 # Variable fonts -----------------------------
 
-[SourceSerifVariable-Roman]
-    f=Source Serif Variable
+[SourceSerif4Variable-Roman]
+    f=Source Serif 4 Variable
     s=Roman
 
-[SourceSerifVariable-Italic]
-    f=Source Serif Variable
```

Comment:
I remember it was decided to keep the name be Source Serif Variable (not 4). I could go either way, but want to make sure this is okay.

Reply:
Yes, it can be either way. The existing font that's installed with apps has the same name so to avoid conflicts and avoid manually deleting those hidden files I changed the name to add 4. For the real final release it can remove the 4 and replace the old font everywhere.